dataactbroker/handlers/accountHandler.py

```diff
@@ -137,6 +144,117 @@ def login(self,session):
         return self.response
 
 
+    def max_login(self,session):
+        """
+
+        Logs a user in if their password matches
+
+        arguments:
+
+        session  -- (Session) object from flask
+
+        return the reponse object
+
+        """
+        try:
+            safeDictionary = RequestDictionary(self.request)
+
+            # Obtain POST content
+            ticket = safeDictionary.getValue("ticket")
+            service = safeDictionary.getValue('service')
+            parent_group = CONFIG_BROKER['parent_group']
+
+            # Call MAX's serviceValidate endpoint and retrieve the response
+            url = CONFIG_BROKER['cas_service_url'].format(ticket, service)
+            max_xml = requests.get(url).content
+            max_dict = xmltodict.parse(max_xml)
+
+            if not 'cas:authenticationSuccess' in max_dict['cas:serviceResponse']:
+                raise ValueError("You have failed to login successfully with MAX")
+
+            # Grab the email and list of groups from MAX's response
+            email = max_dict['cas:serviceResponse']['cas:authenticationSuccess']['cas:attributes']['maxAttribute:Email-Address']
```

Comment:
(I'm all for not wrapping this up in additional error handling if the service says the info should be there, am just asking because I haven't read the relevant MAX docs)